test/internals/handleRequest.test.js

```diff
@@ -183,8 +183,8 @@ test('request should be defined in onSend Hook on options request with content t
       }
     }, (err, response, body) => {
       t.error(err)
-      // a 415 error is expected because of missing content type parser
-      t.strictEqual(response.statusCode, 415)
+      // Body parsing skipped, so no body sent
+      t.strictEqual(response.statusCode, 200)
```

Comment:
Is this a breaking change?

Reply:
To be safe we could consider it a breaking change, but I don't think it is. There isn't a body being sent with the request so I'd consider the old `415 Unsupported Media Type` response to be a bug.

Reply:
I agree. @mcollina?

Reply:
Agreed